README.md

```diff
@@ -33,12 +33,14 @@ After you've created a new plugin directory, update the `index.md`, `plugin.js`,
 Feel free to add additional information to your plugin directory, such as we did with `remote-explorer`.
 
 ## Contribution Guidelines
-
+- Comments on top of the script explaining what is is and how to use it
 - Has to have screenshot, ideally with result of action and or the ui, should to be ~20kb in size unless you really need more
 - Check desctructors cleanup all constructors, delete all new, reset all listeners and timers
```

Comment:
Can we update `desctructors` to `destructors` please?

Reply:
Nice catch, in future remember you can click the insert suggestion button and just change it and then I can just click a merge button
```suggestion
- Check destructors cleanup all constructors, delete all new, reset all listeners and timers
```